packages/fxa-settings/src/components/BentoMenu/index.tsx

```diff
@@ -27,18 +27,24 @@ export const BentoMenu = () => {
   const dropDownId = 'drop-down-bento-menu';
   const iconClassNames = 'inline-block w-5 -mb-1 ltr:pr-1 rtl:pl-1';
   const { l10n } = useLocalization();
+  const bentoMenuTitle = l10n.getString(
+    'bento-menu-title',
+    null,
+    'Firefox Bento Menu'
+  );
 
   return (
     <div className="relative self-center flex" ref={bentoMenuInsideRef}>
       <button
         onClick={toggleRevealed}
         data-testid="drop-down-bento-menu-toggle"
-        title={l10n.getString('bento-menu-title', null, 'Firefox Bento Menu')}
+        title={bentoMenuTitle}
+        aria-label={bentoMenuTitle}
         aria-expanded={isRevealed}
-        aria-controls={dropDownId}
+        aria-haspopup={true}
         className="rounded p-1 w-7 mx-2 border-transparent hover:bg-grey-200 transition-standard desktop:mx-8"
       >
-        <BentoIcon className="cursor-pointer" />
+        <BentoIcon />
```

Comment:
Why take this class off, is it `cursor-pointer` by default (I didn't check into the branch)?

Reply:
Its parent is a `button` that we already apply the style `cursor: pointer;` to, making it unnecessary to add.